packages/elements-core/.storybook/preview.jsx

```diff
@@ -13,7 +13,9 @@ const ThemeProvider = (Story, context) => {
     subscribeTheme({ mode: theme === 'dark' ? 'dark' : 'light' });
   }, [subscribeTheme, theme]);
   return (
-    <Story {...context} />
+    <Box style={{maxWidth: 1200, }} mx='auto' py={10}>
```

Comment:
```suggestion
    <Box style={{ maxWidth: 1200 }} mx='auto' py={10}>
```
nit, but my eyes always hurt when I see such stuff haha

Reply:
Ha - I specifically saved without auto-formatting because I didn't want to increase the diff. I'll just format the file, I agree it hurts my eyes too.